.github/workflows/uninstall_install_sandbox.yaml

```diff
@@ -0,0 +1,66 @@
+name: Uninstall and then re-install Traction Sandbox Instance
+
+on:
+  schedule:
+    - cron: '0 9 1,15 * *'
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  uninstall_sandbox:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Run Your Job
+        run: |
+          echo "This job runs every day at 2:00 AM PST"
+
+      - name: Authenticate and set context
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT_SERVER }}
+          openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
+          namespace: ${{ secrets.OPENSHIFT_NAMESPACE }}
+      
+      - name: Uninstall Traction via Helm
+        continue-on-error: true
+        run: |
+          helm uninstall traction-sandbox -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=20m
+
+      - name: Remove Traction Openshift Objects
+        continue-on-error: true 
+        run: |
+          oc delete -n ${{ secrets.OPENSHIFT_NAMESPACE }} all,secret,pod,networkpolicy,configmap,pvc --selector "app.kubernetes.io/instance"=traction-sandbox
```

Comment:
I think deleting everything - rather than just `secret,configmap,pvc` is fine here since we are trying to reset everything. @i5okie any concern with this?